### PR TITLE
Fixes #135 - Properly store Association.secret on Python 3

### LINKED ISSUE #135: Error "Incorrect padding" when using OpenID
Initially reported as https://github.com/python-social-auth/social-core/issues/180

On Python 3, the secret obtained from the remote server might be saved with the binary prefix "b'secret'" in the association.secret fields, which then lead later to the "Incorrect padding" exception when trying to use that secret.

Replacing `assoc.secret = base64.encodestring(association.secret)` by `assoc.secret = base64.encodestring(association.secret).decode()` in `storage.py` seems to solve the issue.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

**Comment by claudep** [comment]
@omab Anything missing in this PR?

File: social_django/storage.py [hunk 1]
```diff
@@ -152,7 +152,7 @@ def store(cls, server_url, association):
         except cls.DoesNotExist:
             assoc = cls(server_url=server_url,
                         handle=association.handle)
-        assoc.secret = base64.encodestring(association.secret)
+        assoc.secret = base64.encodestring(association.secret).decode()
```
Comment (pjdelport) [inline social_django/storage.py]: Since Python 3.1, This should be `base54.encodebytes` to avoid a `DeprecationWarning`.

Comment (nijel) [reply social_django/storage.py]: But that does not work on Python 2...

**Comment by atodorov** [comment]
ping, any update here ?

File: social_django/storage.py (hunk 1 above)
Comment (atodorov) [reply social_django/storage.py]: @claudep can you if/else or try/except depending on the version of Python this is running on ? 

@reviewers: if this is the only blocker to including this patch I will be happy to cherry-pick that and apply a change for Python 2 (I am running on Python 3 but I need this fix). Let me know how you want to proceed.

Comment (claudep) [reply social_django/storage.py]: Isn't it `encodebytes` which is not Python 2 compatible? Frankly, I'm tired contributing to projects with no response from maintainers. That's a shame.

**Comment by omab** [comment]
Merged including the `encodebytes` change.

**Pull request merged**